portality/templates/publisher/metadata.html

```diff
@@ -28,13 +28,20 @@ <h4 class="red">There is a problem with the submitted form.</h4>
                         <button type="submit" id="remove_{{subfield.short_name}}" name="remove_{{subfield.short_name}}" class="btn btn-danger remove_button">
                           <i class="glyphicon glyphicon-remove"></i>
                         </button>
+                        {% for error in subfield.errors %}
+                            <ul class="errors">
+                                <li>{{ subfield.errors.get(error)[0] }}</li>
+                            </ul>
+                        {% endfor %}
```

Comment:
Same comment as above here - won't you end up with multiple ul elements each with a single li inside them?